Benchmarks/Benchmarks/NIOPosixBenchmarks/Benchmarks.swift

```diff
@@ -39,28 +39,26 @@ let benchmarks = {
     // This benchmark is only available above 5.9 since our EL conformance
     // to serial executor is also gated behind 5.9.
     #if compiler(>=5.9)
-// In addition this benchmark currently doesn't produce deterministic results on our CI
-// and therefore is currently disabled
-//    Benchmark(
-//        "TCPEchoAsyncChannel",
-//        configuration: .init(
-//            metrics: defaultMetrics,
-//            timeUnits: .milliseconds,
-//            scalingFactor: .mega,
-//            setup: {
-//                swiftTaskEnqueueGlobalHook = { job, _ in
-//                    eventLoop.executor.enqueue(job)
-//                }
-//            },
-//            teardown: {
-//                swiftTaskEnqueueGlobalHook = nil
-//            }
-//        )
-//    ) { benchmark in
-//        try await runTCPEchoAsyncChannel(
-//            numberOfWrites: benchmark.scaledIterations.upperBound,
-//            eventLoop: eventLoop
-//        )
-//    }
+    Benchmark(
+        "TCPEchoAsyncChannel",
+        configuration: .init(
+            metrics: defaultMetrics,
+            timeUnits: .milliseconds,
+            scalingFactor: .mega,
```

Comment:
If you update to Benchmark `1.22.0` you can now add some slack to the p90 check for malloc counts, this should do the trick (allowing for a +-100 mallocs);

```suggestion
            scalingFactor: .mega,
            thresholds: [.mallocCountTotal : .init(absolute: [.p90 : 100])],
```